src/components/CurrencyInput.tsx

```diff
@@ -38,12 +38,20 @@ const CurrencyInput: FunctionComponent<Props> = ({
   padZeros = defaultProps.padZeros,
   size,
   value,
-  onChange: onAccept,
+  onChange,
   ...props
 }: Props) => {
   const inputClassNames = classnames('form-control', inputProps && inputProps.className);
+  const onAccept = (val: string, mask: IMask.InputMask<IMask.MaskedNumberOptions>) => {
+    if (onChange) {
+      const ev = new Event('change') as unknown as React.ChangeEvent<HTMLInputElement>;
+      // @ts-ignore
```

Comment:
Needed a couple ts-ignore comments because the imask typings seem to have some mistakes/stuff missing. Can look into fixing it later.